Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -57,7 +61,6 @@ public class OpeningBreakoutAlgorithm : QCAlgorithm
         private const string Symbol = "SPY";
 
         // plotting and logging control
-        private const bool EnablePlotting = true;
         private const bool EnableOrderUpdateLogging = false;
```

Comment:
it is used at line 380 ( if (LiveMode || orderEvent.Status.IsFill() || EnableOrderUpdateLogging)), I didn't want to touch that if you want I can update